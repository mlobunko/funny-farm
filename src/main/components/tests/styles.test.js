import React from 'react';
import { shallow } from 'enzyme';
import 'enzyme-to-json';

import {
  StyledAnimals,
  StyledContentComponent,
  StyledImageRotate,
  StyledRotate,
  StyledRow
} from '../styles';

describe('StyledAnimals', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledAnimals />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledContentComponent', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledContentComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledImageRotate', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledImageRotate />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledRotate', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledRotate />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledRow', () => {
  it('renders correctly with props.top', () => {
    const wrapper = shallow(<StyledRow top />);
    expect(wrapper).toMatchSnapshot();
  });

  it('renders correctly without props.top', () => {
    const wrapper = shallow(<StyledRow />);
    expect(wrapper).toMatchSnapshot();
  });
});
