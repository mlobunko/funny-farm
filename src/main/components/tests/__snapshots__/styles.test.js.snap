// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledAnimals renders correctly 1`] = `
<div
  className="sc-ifAKCX jOcMuq"
/>
`;

exports[`StyledContentComponent renders correctly 1`] = `
<div
  className="sc-bxivhb bfBbdN"
/>
`;

exports[`StyledImageRotate renders correctly 1`] = `
<div
  className="sc-bwzfXH jLakDY"
/>
`;

exports[`StyledRotate renders correctly 1`] = `
<div
  className="sc-htpNat eJGrvs"
/>
`;

exports[`StyledRow renders correctly with props.top 1`] = `
<div
  className="sc-bdVaJa dRvlCK"
/>
`;

exports[`StyledRow renders correctly without props.top 1`] = `
<div
  className="sc-bdVaJa fJksTZ"
/>
`;
